src/scss/vendors/_vendors.scss

```diff
@@ -1 +1,34 @@
 // Vendor-specific styles.
+
+//UIO
+.flc-toc-tocContainer {
```

Comment:
Thanks for trying to fix the "table of content" styling here. The problem is it doesn't fix issues thoroughly, for example, this css sets the background color of toc div to white but leaves a gap between toc div and main content div showing background image. 

It might be better to remove the toc fix from this pull request. Once we decide what should be fixed when toc is enabled, a separate ticket can be created to address all toc related issues thoroughly.